CI-CD/AzureDevOps/azure-pipelines_ECR Image scan.yml

```diff
@@ -0,0 +1,55 @@
+# Generic example for scanning docker images with the WhiteSource Unified Agent from Amazon Elastic Container Registry
+
+# Add AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY as environment variables for the AWS CLI configuration/authentication (https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html).
+# Add WS_APIKEY & WS_USERKEY as environment variables for the WhiteSource authentication.
+# Adjust variables to match your Image URI.
+
+trigger:
+  branches:
+    include:
+    - main
+variables:
+- name: aws_region
+  value: 'us-east-1'
+- name: aws_account_id
+  value: '<aws accoun id>'
+- name: image
+  value: '<image name>'
+stages:
+- stage: __default
+  jobs:
+  - job: Job
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - task: CmdLine@2
+      displayName: 'Pull Image $(image) from ECR'
+      inputs:
+        script: |
+          aws configure set aws_access_key_id $(AWS_ACCESS_KEY_ID)
+          aws configure set aws_secret_access_key $(AWS_SECRET_ACCESS_KEY)
+          aws configure set region $(aws_region)
+          aws ecr get-login-password --region $(aws_region) | docker login --username AWS --password-stdin $(aws_account_id).dkr.ecr.$(aws_region).amazonaws.com
+    - task: CmdLine@2
+      env:
+        WS_APIKEY: $(APIKEY)
+        WS_USERKEY: $(USERKEY)
+        WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+        WS_PRODUCTNAME: $(System.TeamProject)
+        WS_PROJECTNAME: doesnotmatter
+        WS_DOCKER_SCANIMAGES: true
+        WS_DOCKER_INCLUDES: ".*$(image).*"
+        WS_DOCKER_AWS_ENABLE: true
+        WS_DOCKER_PULL_ENABLE: true
+        WS_DOCKER_PULL_IMAGES: ".*$(image).*"
+        WS_DOCKER_AWS_REGISTRYID: $(aws_account_id)
+        WS_DOCKER_LAYERS: true
+        WS_DOCKER_PROJECTNAMEFORMAT: repositoryNameAndTag
+        WS_FILESYSTEMSCAN: false
```

Comment:
add archive extraction as shown in https://github.com/whitesource-ft/ws-examples/blob/main/Generic/UA-Docker-Image.sh

Reply:
Added archive extraction